docs/configuration.asciidoc

```diff
@@ -133,6 +133,22 @@ function nodeSelector (connections) {
   return connections[index]
 }
 ----
+
+|`generateRequestId`
+a|`function` - function for generate the request id of every request. +
```

Comment:
> By default we call the internal generateRequestId() function to create another function with a closure scope. We don't do this for the generateRequestId() passed in options, do we want to?

Nope, I expect that the user passes me a function to generates unique ids, if this function requires a closure, the user will take care of that.

Reply:
What do you think about
> I think it would be really useful to pass this function the parameters received by the client so that they can be used to derive the request id, but maybe that's a bad idea as it encourages passing undocumented parameters...

Reply:
Sorry! Forgot about this one.
Can you make a code example?

Reply:
Now I got you, yes we can do that :)

Reply:
```
function genReqId (requestParams, requestOptions) {
 ...
}
```